app.py

```diff
@@ -176,7 +178,8 @@
     cluster=ecs_stack.cluster,
     props=apex_props,
     load_balancer=load_balancer_stack.alb,
-    certificate_arn=environment_variables["CERTIFICATE_ARN"],
+    certificate_account_id=environment_variables["CERTIFICATE_ACCOUNT_ID"],
+    certificate_resource_id=environment_variables["CERTIFICATE_RESOURCE_ID"],
```

Comment:
I don't like that this solution requires setting tags because it's kinda a jenky work around.